src/main/java/ch/uzh/ifi/hase/soprafs24/service/RatingService.java

```diff
@@ -23,30 +19,27 @@
 public class RatingService {
     private final Logger log = LoggerFactory.getLogger(RatingService.class);
     private final RatingRepository ratingRepository;
```

Comment:
This has been refactored so that only the RatingRepository is called. Any call related to users is done through the UserService, not the userRepository anymore.